langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -39,6 +44,20 @@ public isolated function length(map<any|error> m) returns int =@java:Method {
 
 # Returns an iterator over a map.
 #
+# ```ballerina
+# map<string> vowels = {a: "A", e: "E", i: "I", o: "O", u: "U"};
+#
+# // iterator() returns an iterator object with the `next()` method.
+# var vowelsIter = vowels.iterator();
```

Comment:
Let's not use `var` here. Can you also check the array examples in https://github.com/ballerina-platform/ballerina-lang/pull/38892/files. I think the array and map examples should be similar, where applicable. If there's something you think needs to change there we can revisit too.